test/powershell/Language/Scripting/MyInvocation.Tests.ps1

```diff
@@ -0,0 +1,50 @@
+﻿Describe 'Testing of MyInvocation' -Tags "CI" {
+    It 'Script file' {
+
+        $MyInvocation.Line.ToLower().IndexOf('$null = & $scriptblock @parameters') | Should BeGreaterThan -1
```

Comment:
This test is sensitive to code in Pester because you're matching a line of code from Pester. This is fragile, you should change it to match something in this file, e.g.
```powershell
(<#FOO#> & { $MyInvocation.Line }) | Should Match 'FOO'
```